src/main/java/org/tikv/common/operation/PDErrorHandler.java

```diff
@@ -74,6 +75,10 @@ public boolean handleResponseError(BackOffer backOffer, RespT resp) {
 
   @Override
   public boolean handleRequestError(BackOffer backOffer, Exception e) {
+    // store id is not found
+    if (e instanceof StatusRuntimeException && e.getMessage().contains("invalid store ID")) {
```

Comment:
can we directly check whether the instance is of `InvalidStoreException`?

Reply:
BTW, should the error message be `Invalid storeId`?

Reply:
no, the exception tikv returns is a grpc exception, it is a StatusRuntimeException that only contains "invalid store id xx" information in its message.